templates/apollo-starter-ts/package.json

```diff
@@ -20,7 +20,8 @@
     "graphql-tag": "2.10.1",
     "ioredis": "4.14.0",
     "knex": "0.19.1",
-    "pg": "7.12.0"
+    "pg": "7.12.0",
```

Comment:
I think that is the smell that we want to address. We do not want every template 
to have access to library that it might not use. Additionaly - template will have postgress related code. I remember we talked about this before so we might drop some new requirements or create new issue to cover this problem

Reply:
I think we may have to inject package.json too, like other files. And the only config it depends upon can be imported from the config.

Reply:
Or have list of depenencies that will be extecuted on top of npm instal. Lets create issue to chat about it. I think having extra dependencies could be cool

Reply:
Oh yes I didn't think of that

Reply:
The only files that were postgres specific were `config.ts` and `db.ts`. Used `resolveJsonModule` in the recent commit to read config from the `config.json` file